test/ci/kokoro/linux/run_integ_tests.sh

```diff
@@ -80,7 +80,7 @@ init_python
 update_submodules
 
 # Check that we're using the correct config
-gsutil version -l
+python "$GSUTIL_ENTRYPOINT" -l
```

Comment:
Whoops - you left out the `version` before the `-l` here.

Reply:
Oh dang, thanks for catching that! Fixed :)